src/core/bonded_interactions/bonded_coulomb.hpp

```diff
@@ -44,34 +44,30 @@ int bonded_coulomb_set_params(int bond_type, double prefactor);
  *  @param[in]  q1q2      Product of the particle charges.
  *  @param[in]  iaparams  Interaction parameters.
  *  @param[in]  dx        %Distance between the particles.
- *  @param[out] force     Force.
- *  @retval false
  */
-inline bool calc_bonded_coulomb_pair_force(
-    double const q1q2, Bonded_ia_parameters const *const iaparams,
-    Utils::Vector3d const &dx, Utils::Vector3d &force) {
+inline boost::optional<Utils::Vector3d>
+calc_bonded_coulomb_pair_force(double const q1q2,
```

Comment:
why is there a ```calc``` before the force calculating function's names but not for the energy kernels? Maybe we should drop the prefix.

Reply:
I've now removed the prefix in force/energy kernels